python/ray/tune/tests/test_trial_scheduler_pbt.py

```diff
@@ -64,6 +64,10 @@ def load_checkpoint(self, path):
                 with open(path, "rb") as fp:
                     self.large_object, self.iter, self.a = pickle.load(fp)
 
+            def reset_config(self, new_config):
+                self.a = new_config["a"]
+                return True
+
```

Comment:
Why do we need this? It seems the that `reuse_actors=False`?

Reply:
pbt uses `reset_trial` directly instead of going through `reuse_actors`.